kevm-pyk/src/kevm_pyk/utils.py

```diff
@@ -71,23 +70,15 @@ def kevm_prove(
     kprove: KProve,
     proof: Proof,
     kcfg_explore: KCFGExplore,
-    save_directory: Path | None = None,
     max_depth: int = 1000,
     max_iterations: int | None = None,
-    workers: int = 1,
     break_every_step: bool = False,
     break_on_jumpi: bool = False,
     break_on_calls: bool = True,
     implication_every_block: bool = False,
     is_terminal: Callable[[CTerm], bool] | None = None,
     extract_branches: Callable[[CTerm], Iterable[KInner]] | None = None,
     same_loop: Callable[[CTerm, CTerm], bool] | None = None,
-    bmc_depth: int | None = None,
```

Comment:
Maybe not passing this one along is a bug?

Reply:
I think the proof has already been constructed at this point, as the second parameter, and it containts the `bmc_depth` if it's a `APRBMCProof`.